src/Symfony/Component/Translation/Dumper/JsonFileDumper.php

```diff
@@ -23,9 +23,23 @@ class JsonFileDumper extends FileDumper
     /**
      * {@inheritdoc}
      */
+    protected function formatCatalogue(MessageCatalogue $messages, $domain, array $options = array())
+    {
+        if (isset($options['json_encoding'])) {
```

Comment:
but can we take on https://github.com/symfony/symfony/pull/14750#issuecomment-105272043 ?